LiveLines/ServiceModuleExtension.cs

```diff
@@ -14,14 +16,15 @@ public static class ServiceModuleExtension
     public static IServiceCollection AddServices(this IServiceCollection services)
     {
         // TODO: I should be able to make all of the concrete implementations internal
-        
+
         return services
             .AddSingleton<IDatabaseCommandExecutor, PostgresCommandExecutor>()
             .AddSingleton<IUserService, UserService>()
             .AddSingleton<IUserStore, UserStore>()
             .AddSingleton<ILinesService, LinesService>()
             .AddSingleton<ILinesStore, LinesStore>()
             .AddSingleton<ISongStore, SongStore>()
-            .AddSingleton<ISongService, SongService>();
+            .AddSingleton<ISongService, SongService>()
+            .AddSingleton<IStreakService, StreakService>();
```

Comment:
As we discussed earlier with commas, you may want to have the semi colon on the next line down